docs/custom-authentication.md

```diff
@@ -6,6 +6,7 @@ Netlify CMS is meant to be platform agnostic, so we're always looking to expand
 | Author     | Supported Git hosts       | Languages | Link                                                                |
 |------------|---------------------------|-----------|---------------------------------------------------------------------|
 | @vencax    | GitHub, GitHub Enterprise | Node.js   | [Repo](https://github.com/vencax/netlify-cms-github-oauth-provider) |
+| @igk1972   | GitHub, GitHub Enterprise, Bitbacket, Gitlab | Go | [Repo](https://github.com/igk1972/netlify-cms-oauth-provider-go) |
```

Comment:
I wonder if we should add a note below this table saying that BitBucket and GitLab are not yet supported in the CMS? Or just let people figure that out on their own? @verythorough?

Reply:
Little step to the future. Main keyword "...**not yet**...".

Edit line in docs/custom-authentication.md ?

Reply:
If it doesn't actually support those hosts, I'd say remove them for now.